spec/requests/api/v1/ticket_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'as a registered user', :type => :request do
+  describe 'when i go to the ticket dashboard' do
+    before :each do
+      User.create(id:1, name:"jennica", email:"jennica.stiehl@gmail.com", password_digest:"password", role:0)
+      ResourceType.create(id: 1, category:"vehicle", company:"Brew Bears")
+      ResourceType.create(id: 2, category:"equipment", company:"Brew Bears")
+      Resource.create(id: 1, resource_type_id:1, name:"van", cost:"75000", user_id:1)
+      Resource.create(id: 2, resource_type_id:1, name:"car", cost:"25000", user_id:1)
+      Resource.create(id: 3, resource_type_id:2, name:"bottler", cost:"5000", user_id:1)
+      Ticket.create(table_key: 1, table_name: "Resources",user_id:1, notes:"needs oil change", priority:1)
+      Ticket.create(table_key: 2, table_name: "Resources",user_id:1, notes:"needs new transmission", priority:3)
```

Comment:
This table is a table for each different note correct?

Reply:
This is the table where all of the tickets live.